data.sql

```diff
@@ -75,3 +75,32 @@ SET owner_id =
         WHEN name IN ('Angemon', 'Boarmon') THEN 5
         ELSE NULL
      END);
+
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 40, '2000-4-23'),('Maisy', 26, '2019-1-17'),('Stephanie Mendez', 64, '1981-5-4'),('Jack Harkness', 38, '2008-6-8');
+
+INSERT INTO specializations (vet_id, species_id)
+vet_clinic-# VALUES (1, 1), (3, 2), (3, 1), (4, 2);
```

Comment:
- [ ] It would be better if we remove the `#`, every other text that comes after it will be interpreted as comments, hence the `VALUES` are not recorded in the database